src/store/messages/saga.ts

```diff
@@ -326,7 +326,7 @@ export function* deleteMessage(action) {
   const existingMessageIds = yield select(rawMessagesSelector(channelId));
   const fullMessages = yield select((state) => denormalize(existingMessageIds, state));
 
-  const messageIdsToDelete = fullMessages.filter((m) => m.rootMessageId === messageId.toString()).map((m) => m.id); // toString() because message ids are currently a number
+  const messageIdsToDelete = fullMessages.filter((m) => m.rootMessageId === messageId).map((m) => m.id);
```

Comment:
Having said that, maybe we don't care. lol

Reply:
Appeared to work when I tried it with sendbird. Just to double check..

I've changed the call in sendbird to not swallow the error, and the type to string (because `deleteMessageByRoomId`'s `messageId` type is now string as suggested :
```
async deleteMessageByRoomId(roomId: string, messageId: string): Promise<void> {
    await del<any>(`/chatChannels/${roomId}/message`).send({ message: { id: messageId } });
  }
```

And, as you commented here, remove the `toString` 🤔

Reply:
I didn't say remove `toString`. I said the interface to the generic chat client call should take a string and not a number.

Reply:
Sorry, my point here was...there's a reason that saga code had the `toString` in that filter. It has nothing to do with the api call. That filter is figuring out which messages to delete. If you remove that `toString` there then it won't filter correctly.

Reply:
In the end... it doesn't matter. We're killing sendbird anyway so just leave it. lol